mas-foundation/src/androidTest/java/com/ca/mas/foundation/MASSecureSharedStorageTest.java

```diff
@@ -29,8 +30,11 @@
 
     private final String accountName = "testNameEncrypted";
     private final String accountNameb = "testNameNoEncrypted";
+    private final int TIME = 3500;
```

Comment:
@witrisna @spetrov this time measurement was done using both simulator and a real device, but I think it can change between devices, so suggestions are welcome :)